test/src/test/testtool/MessageStoreSenderAndListenerPeek/scenario01/md-select-result.xml

```diff
@@ -1,52 +1,58 @@
-    <field name="EXPIRYDATE" type="TIMESTAMP" columnDisplaySize="26" precision="26" scale="6" isCurrency="false" columnTypeName="TIMESTAMP" columnClassName="java.sql.Timestamp"/>
-    <field name="LABEL" type="VARCHAR" columnDisplaySize="100" precision="100" scale="0" isCurrency="false" columnTypeName="VARCHAR" columnClassName="java.lang.String"/>
-  </fielddefinition>
-  <rowset>
-    <row number="0">
-      <field name="MESSAGEKEY">161589</field>
-      <field name="TYPE">A</field>
-      <field name="SLOTID">IFSA://NLIBIS4TEST/MessageStoreListener</field>
-      <field name="HOST">LPZWNL000030227</field>
-      <field name="MESSAGEID">Test Tool correlation id</field>
-      <field name="CORRELATIONID">Test Tool correlation id</field>
-      <field name="MESSAGEDATE">2020-02-06 11:08:54</field>
-      <field name="COMMENTS">log</field>
-      <field name="MESSAGE">&lt;request action="store"&gt;
+<manageDatabaseRLY>
+	<completionInformation>
+		<returnCode>OK</returnCode>
+	</completionInformation>
+	<results>
+		<result item="1">
+			<result>
+				<fielddefinition>
+					<field name="MESSAGEKEY" type="NUMERIC" columnDisplaySize="39" precision="38" scale="0" isCurrency="true" columnTypeName="NUMBER" columnClassName="java.math.BigDecimal"/>
+					<field name="TYPE" type="CHAR" columnDisplaySize="1" precision="1" scale="0" isCurrency="false" columnTypeName="CHAR" columnClassName="java.lang.String"/>
+					<field name="SLOTID" type="VARCHAR" columnDisplaySize="100" precision="100" scale="0" isCurrency="false" columnTypeName="VARCHAR2" columnClassName="java.lang.String"/>
+					<field name="HOST" type="VARCHAR" columnDisplaySize="100" precision="100" scale="0" isCurrency="false" columnTypeName="VARCHAR2" columnClassName="java.lang.String"/>
+					<field name="MESSAGEID" type="VARCHAR" columnDisplaySize="100" precision="100" scale="0" isCurrency="false" columnTypeName="VARCHAR2" columnClassName="java.lang.String"/>
+					<field name="CORRELATIONID" type="VARCHAR" columnDisplaySize="256" precision="256" scale="0" isCurrency="false" columnTypeName="VARCHAR2" columnClassName="java.lang.String"/>
+					<field name="MESSAGEDATE" type="TIMESTAMP" columnDisplaySize="11" precision="0" scale="6" isCurrency="false" columnTypeName="TIMESTAMP" columnClassName="oracle.sql.TIMESTAMP"/>
+					<field name="COMMENTS" type="VARCHAR" columnDisplaySize="1000" precision="1000" scale="0" isCurrency="false" columnTypeName="VARCHAR2" columnClassName="java.lang.String"/>
+					<field name="MESSAGE" type="BLOB" columnDisplaySize="4000" precision="-1" scale="0" isCurrency="false" columnTypeName="BLOB" columnClassName="oracle.jdbc.OracleBlob"/>
+					<field name="EXPIRYDATE" type="TIMESTAMP" columnDisplaySize="11" precision="0" scale="6" isCurrency="false" columnTypeName="TIMESTAMP" columnClassName="oracle.sql.TIMESTAMP"/>
```

Comment:
Wat is er nu precies veranderd behalve dat het java.sql nu oracle.jdbc is?

Reply:
`<field name="COMMENTS">log</field>` 
is geworden:
`<field name="COMMENTS">exitState [success], result [String: &lt;request action="store"&gt;
	&lt;test&gt;hello73&lt;/test&gt;
&lt;/request&gt;]</field>`